### PR TITLE
Added interface building limits

### PR DESCRIPTION
**Pull request by Hilbrand**
This way code that checks limits can be independently being developed from OPS and ADMS.

This change introduces the ADMSLimits file in this project.

### CODE REVIEW

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    return false;
+  }
+
+  int buildingDigitsPrecision();
+
+  double buildingHeightMinimum();
+  double buildingHeightMaximum();
+  double buildingHeightDefault();
+
+
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java:35]: Perhaps a whiteline too much?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  double buildingWidthMinimum();
+  double buildingWidthMaximum();
+  double buildingWidthDefault();
+
+  double buildingLengthMinimum();
+  double buildingLengthMaximum();
+  double buildingLengthDefault();
+
+  double buildingOrientationMinimum();
+  double buildingOrientationMaximum();
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java:45]: Not sure if it makes much sense to have a min/max for orientation? It's something we determine/define ourselves based on the geometry that is supplied, and not something that the user has to enter.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  int buildingDigitsPrecision();
+
+  double buildingHeightMinimum();
+  double buildingHeightMaximum();
+  double buildingHeightDefault();
+
+
+  double buildingWidthMinimum();
+  double buildingWidthMaximum();
+  double buildingWidthDefault();
```
Comment (BertScholten) [inline source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java:38]: Not sure how these defaults are used: our application uses geometry, so we don't really need a default? Or is it possible in ADMS to define a building without defining width/length, and these defaults are used when reading an ADMS specific file?

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java (hunk 2 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java]: There is a difference between OPS (179) and ADMS (360). But we don't use it anywhere. So I'll remove it.

File: source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java (hunk 3 above)
Comment (Hilbrand) [reply source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/building/BuildingLimits.java]: Hmm. Can be removed. I copied them from ADMS specification. But we don't use it anyway, so I'll remove them.

**Review by BertScholten — APPROVED** [review]
LGTM

**Pull request merged**